docs/interactions/structured-messages/rich-link.md

```diff
@@ -55,7 +55,18 @@ Primary parameters are used by default, however, some parameters are unique or o
 | **`structured_content.title`** | String | Truncated to 200 characters. |
 | **`structured_content.attachment_id`** | String | Supports jpg, jpeg, png, and mp4 formats. Supports private attachments. Supports up to 300MB. [Upload attachments](../../../basics/uploads) for you own custom images. |
 | **`structured_content.url_text`** | String | Ignored property. |
-| **`structured_content.subtitle`** | String | Ignored property. |
+| **`structured_content.subtitle`** | String | Ignored property (except for iMessage App) |
+
+### iMessage App
+
+Rich Link structured messages can be used to create iMessage apps. To do so, you need to set the `structured_content.url` to a query string starting with `?`. And use the following properties:
+
+| API Property | Type | Description |
+|-|-|-|
+**`app_id`** | String | The App Store identifier of the iMessage app. |
```

Comment:
Please add the `structured.` prefix on the 4 properties so that we're consistent.